docker/nginx/proxito.conf

```diff
@@ -20,10 +20,10 @@ server {
     }
 
     # Sendfile support to serve the actual files that Proxito has specified
-    location /proxito/ {
+    location /proxito/devstoreaccount1/media/ {
```

Comment:
This change is needed because we are using `storage.url` in the `ServeDocsBase` instead of just calling `os.path.join` to generate the path.